python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -260,6 +260,14 @@ def summary_str(self, resource_manager: ResourceManager) -> str:
         queued = self.num_queued() + self.op.internal_queue_size()
         active = self.op.num_active_tasks()
         desc = f"- {self.op.name}: {active} active, {queued} queued"
+        if (
+            self.op._in_task_submission_backpressure
+            or resource_manager.op_resource_allocator.max_task_output_bytes_to_read(
+                self.op
+            )
+            == 0
```

Comment:
Can we also use a flag to store the latest state for max_task_output_bytes_to_read ? because:
1.  max_task_output_bytes_to_read is not always used (only when the reservation policy is used). 
2. calling .max_task_output_bytes_to_read has overheads. 
3. decoupling the logic.